packages/vow/src/index.js

```diff
@@ -5,3 +5,6 @@ export { VowShape, toPassableCap } from './vow-utils.js';
 
 // eslint-disable-next-line import/export
 export * from './types.js';
+
+// XXX re-exporting the Remote type for back-compat
+export * from '@agoric/internal/src/types.js';
```

Comment:
Can we limit the re-exports only to what is actually needed?

```suggestion
// XXX re-exporting the Remote type for back-compat
export {Remote} from '@agoric/internal/src/types.js';
```

Reply:
no, unfortunately. there aren't actually any values exported from that module and that's what that expression would require. the `export *` means that JS sees `{}` but TS sees the types too.